BlogTemplate/Pages/Drafts.cshtml.cs

```diff
@@ -24,7 +24,7 @@ public DraftsModel(BlogDataStore dataStore)
 
         public void OnGet()
         {
-            Posts = _dataStore.GetAllPosts();
+            Posts = _dataStore.GetAllPosts().Where(post => !post.IsPublic);

```

Comment:
That's true, the reason why I changed it back to Posts is that within the drafts, we don't want to further categorize the posts if we don't have to. So within the Drafts page the only Posts are the private posts so it doesn't make much sense to change the name to Private Posts. The same goes for the Index page where all of the Posts should be the posts that are published or public so it adds a little confusion if we call that public posts because some might think that refers to public posts within all of the published posts.

Does this make sense?